WordPress/src/main/java/org/wordpress/android/ui/compose/components/MediaUriPager.kt

```diff
@@ -0,0 +1,165 @@
+    )
+    HorizontalPager(
+        state = pagerState,
+        pageSpacing = 12.dp,
+        pageSize = PageSize.Fixed(IMAGE_SIZE.dp),
+        modifier = Modifier.then(modifier)
+    ) { index ->
+        val uri = mediaUris[index]
+        Box(
+            modifier = Modifier.height(IMAGE_SIZE.dp),
+        ) {
+            MediaUriImage(uri)
+            if (showButton) {
+                ImageButton(uri, onButtonClick)
+            }
+        }
+    }
+}
+
+@Composable
+private fun MediaUriImage(uri: Uri) {
+    val context = LocalContext.current
+    val mimeType = context.contentResolver.getType(uri)
+    if (mimeType?.startsWith("video/") == true) {
+        val imageLoader = ImageLoader.Builder(LocalContext.current)
+            .components {
+                add(VideoFrameDecoder.Factory())
+            }
+            .build()
+        Box {
+            AsyncImage(
+                model = ImageRequest.Builder(context)
+                    .data(uri)
+                    .crossfade(true)
+                    .build(),
+                imageLoader = imageLoader,
+                contentScale = ContentScale.Crop,
+                contentDescription = null,
+                modifier = Modifier
+                    .border(1.dp, MaterialTheme.colorScheme.onBackground.copy(alpha = 0.25f))
+                    .size(IMAGE_SIZE.dp)
+            )
+            Image(
+                imageVector = ImageVector.vectorResource(id = org.wordpress.android.editor.R.drawable.ic_overlay_video),
+                contentDescription = null,
+                modifier = Modifier
+                    .align(Alignment.Center)
+                    .size(36.dp)
+            )
+        }
+    } else {
+        AsyncImage(
+            model = ImageRequest.Builder(context)
+                .data(uri)
+                .crossfade(true)
+                .placeholder(R.color.placeholder)
+                .error(org.wordpress.android.editor.R.drawable.ic_image_failed_grey_a_40_48dp)
+                .build(),
+            contentScale = ContentScale.Crop,
+            contentDescription = null,
+            modifier = Modifier
+                .border(1.dp, MaterialTheme.colorScheme.onBackground.copy(alpha = 0.25f))
+                .size(IMAGE_SIZE.dp)
+        )
+    }
+}
+
+@Composable
+private fun BoxScope.ImageButton(
+    uri: Uri,
+    onButtonClick: (Uri) -> Unit = {},
+) {
+    IconButton(
+        onClick = { onButtonClick(uri) },
+        modifier = Modifier
+            .absoluteOffset(x = (-2).dp, y = (-2).dp)
+            .background(
+                color = MaterialTheme.colorScheme.surface.copy(alpha = 0.7f),
+                shape = RoundedCornerShape(2.dp)
+            )
+            .size(24.dp)
+            .align(Alignment.BottomEnd),
+    ) {
+        Icon(
+            imageVector = Icons.Filled.Close,
+            tint = MaterialTheme.colorScheme.onSurface,
+            contentDescription = stringResource(R.string.remove),
```

Comment:
This description is less helpful when navigating multiple items with TalkBack. It is difficult to discern which item will be removed. We might consider appending the item index to the description. 

I also note that focus was often reset to the dialog close button when attempting to navigate through the remove buttons via TalkBack, which is quite disruptive. Not sure what might be reseting the focus.

Reply:
> This description is less helpful when navigating multiple items with TalkBack...We might consider appending the item index to the description.

That's a very good point. I just made this change

> I also note that focus was often reset to the dialog close button when attempting to navigate through the remove buttons via TalkBack

I wasn't able to reproduce this, but perhaps I'm misunderstanding. Can you describe the steps in more detail?

Reply:
1. Navigate to the feedback form. 
2. Attach five items. 
3. Enable TalkBack. 
4. Swipe from left to right on the screen to move selection to a "remove" button that is hidden off screen.
5. Wait momentarily for the TalkBack selection to reset and move to the dialog close button. 

You'll note at the end of the attached screen recording that the TalkBack selection jumps to the close button in the top-left corner. It appears that scrolling the horizontal pager via moving selection to the next remove button that is off screen creates this effect. 

Does the page rely upon virtualization? Maybe that could be disrupting TalkBack selection. 

<details><summary>Screen recording</summary>


https://github.com/user-attachments/assets/d71e28a5-12c4-40e8-b7e9-57a8678c0f7e



</details>

Reply:
Thanks, David, that was very helpful. It's not easy to reproduce but I was able to do so. I thought maybe it had to do with the pager not laying out all the images, but I forced them to be laid out and the problem still occurred.

I'm going to file a separate issue for this.